### ISSUE TITLE
'Wait for output' steps are broken

### ISSUE THREAD

**Issue by jcarlson** [cucumber/aruba#854]
**Describe the bug**
`ExpectationError` is not defined in https://github.com/cucumber/aruba/blob/main/lib/aruba/cucumber/command.rb#L88-L104

**To Reproduce**
```cucumber
Feature: Reproduce the error
    Scenario: Trouble in Hello Town
        When I run `echo hello town`
        And I wait for stdout to contain "hello world"
        Then the exit status should be 0
```

**Expected behavior**
Test should fail, probably with an `RSpec::Expectations::ExpectationNotMetError`

**Screenshots**
```bash
$ rake
/Users/jcarlson/.rbenv/versions/2.6.8/bin/ruby -S bundle exec cucumber
Feature: Aruba runs correctly

  Scenario: Hello world                          # features/bermuda.feature:3
    When I run `echo hello world`                # aruba-1.1.2/lib/aruba/cucumber/command.rb:3
    Then the output should contain "hello world" # aruba-1.1.2/lib/aruba/cucumber/command.rb:117

  Scenario: Hello again world                      # features/bermuda.feature:7
    When I run `echo hello world`                  # aruba-1.1.2/lib/aruba/cucumber/command.rb:3
    And I wait for stdout to contain "hello world" # aruba-1.1.2/lib/aruba/cucumber/command.rb:100
    Then the exit status should be 0               # aruba-1.1.2/lib/aruba/cucumber/command.rb:277

  Scenario: Trouble in Hello Town                  # features/bermuda.feature:12
    When I run `echo hello town`                   # aruba-1.1.2/lib/aruba/cucumber/command.rb:3
    And I wait for stdout to contain "hello world" # aruba-1.1.2/lib/aruba/cucumber/command.rb:100
      uninitialized constant ExpectationError (NameError)
      features/bermuda.feature:14:in `I wait for stdout to contain "hello world"'
    Then the exit status should be 0               # aruba-1.1.2/lib/aruba/cucumber/command.rb:277

Failing Scenarios:
cucumber features/bermuda.feature:12 # Scenario: Trouble in Hello Town

3 scenarios (1 failed, 2 passed)
8 steps (1 failed, 1 skipped, 6 passed)
0m0.030s
```

**Additional context**
Tested with `aruba@2.0.0` with `cucumber@7.1.0` and `aruba@1.1.2` with `cucumber@6.1.0`

**Comment by jcarlson** [comment]
#407 represents the exact use case that I am running into when I discovered this issue. Given it was closed due to inactivity some ~4 years ago, I think I'm going to have to assume that `aruba` is not well enough supported to continue investing time in to use on my project.

**Comment by mvz** [comment]
Huh, you're writing Aruba off because of something that happened 4 years ago under a different maintainer?

**Comment by mvz** [comment]
Anyway, thanks for the bug report itself. I will look into it.

**Comment by mvz** [comment]
It turns out this step is untested and undocumented and doesn't do what it promises. You should use `Then the output should contain "hello world"` instead.

**Comment by mvz** [comment]
I completely rewrote the step based on a related step and now it does actually wait for output without first stopping the command. This fix will be part of Aruba's next patch release.

**Comment by jcarlson** [comment]
@mvz I apologize for sounding short, and I appreciate that you've taken the time to address this issue.

I think Aruba is a really cool project, and I haven't seen anything else much like it out there for testing command-line tools. I'd really like to make use of it, I've just been frustrated by the on-boarding process.

As far as I am aware, Cucumber doesn't have a concept of "public" vs "private" step definitions. If they are defined in code, they are available to the tests, and many modern IDEs will auto-complete those steps for you as you work. That's how I discovered this [broken] step definition.

I can appreciate that the Aruba documentation is itself a set of tests but, in my opinion, they are hard to read through. There is no table of contents or index, for example, of Step Definitions, something I feel would go a long way to making the project more approachable and discoverable.

My specific use case is a command line tool (built using Thor) which interactively prompts the user for input to gather configuration and writes it to a file. I haven't been able to find a way with the currently released versions of Aruba to assert that a given prompt appears, provide input to it, and then verify the contents of the file.

I'd be willing to give it another try after the next release, but in the mean time, it sounds like Aruba (as released today) simply doesn't offer the capabilities I need to test my app, which is why I stopped trying to use it.

If you have any other suggestions, though, I would be happy to hear them!
